CHANGELOG.md

```diff
@@ -11,14 +11,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Release Notes
+- Nos now publishes the hashtags it finds in your note when you post. This means it works the way you’ve always expected it to work. [#44](https://github.com/verse-pbc/issues/issues/44)
 - Fixed galleries expanding past the width of the screen when there are lots of links or images. [#24](https://github.com/verse-pbc/issues/issues/24)
 - Added support for user setting and displaying pronouns.
 - Added display of website urls for user profiles.
 
 ## [1.0.2] - 2024-11-26Z
 
 ### Release Notes
-- Fix typo in minimum age warning
```

Comment:
Why did this go away?

Reply:
If I had to guess, I'd say it was a merge issue after we released 1.0.2, but I don't know for sure. I'll figure it out.

Reply:
Restored!